config/database.php

```diff
@@ -96,13 +96,17 @@
         'client' => 'predis',
 
         'default' => [
+            'scheme' => env('REDIS_SCHEME', 'tcp'),
+            'path' => env('REDIS_PATH', '/var/run/redis/redis.sock'),
             'host' => env('REDIS_HOST', 'localhost'),
             'password' => env('REDIS_PASSWORD', null),
             'port' => env('REDIS_PORT', 6379),
             'database' => env('REDIS_DATABASE', 0),
         ],
 
         'sessions' => [
+            'scheme' => env('REDIS_SCHEME', 'tcp'),
+            'path' => env('REDIS_PATH', '/var/run/redis/redis.sock'),
```

Comment:
This should probably be changed to `/run/redis/redis.sock` as most modern OSes symlink `/var/run` to `/run`.